wolfssl/internal.h

```diff
@@ -2841,6 +2841,7 @@ struct WOLFSSL_CTX {
     byte        noTicketTls12:1;  /* TLS 1.2 server won't send ticket */
 #endif
 #ifdef WOLFSSL_TLS13
+    byte        maxTicketTls13;   /* maximum number of tickets to send */
```

Comment:
We should not be putting `byte` variables between bits. Please move those up the struct.